packages/ui/src/ui/pages/users/UsersPageEditor/UsersPageEditor.tsx

```diff
@@ -328,6 +345,20 @@ class UsersPageEditor extends React.Component<Props, State> {
                                 ...errors,
                             ],
                         },
+                        {
+                            type: 'tab-vertical',
+                            name: 'password',
```

Comment:
Shouldn't we add `Confirm Password` field to let user be sure the password is correct.